<ui version="4.0" >
 <class>GasChoiceSound</class>
 <widget class="QDialog" name="GasChoiceSound" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choice of a sound</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="lSystem_1" >
       <property name="text" >
        <string>System 1:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="lSystem_3" >
       <property name="text" >
        <string>System 3:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QToolButton" name="tbBrowse_3" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rsc/gasman.qrc" >
         <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leSystem_2" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="lSystem_4" >
       <property name="text" >
        <string>System 4:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leSystem_3" />
     </item>
     <item row="3" column="2" >
      <widget class="QToolButton" name="tbBrowse_4" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rsc/gasman.qrc" >
         <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="lSystem_2" >
       <property name="text" >
        <string>System 2:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leSystem_1" />
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="tbBrowse_2" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rsc/gasman.qrc" >
         <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QToolButton" name="tbBrowse_1" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rsc/gasman.qrc" >
         <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="leSystem_4" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSystem_1</tabstop>
  <tabstop>tbBrowse_1</tabstop>
  <tabstop>leSystem_2</tabstop>
  <tabstop>tbBrowse_2</tabstop>
  <tabstop>leSystem_3</tabstop>
  <tabstop>tbBrowse_3</tabstop>
  <tabstop>leSystem_4</tabstop>
  <tabstop>tbBrowse_4</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../rsc/gasman.qrc" />
 </resources>
 <connections/>
</ui>
